ingestion-beam/Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN mvn package -Dmaven.test.skip=true
 # https://cloud.google.com/dataflow/docs/guides/templates/configuring-flex-templates#changing_the_base_image
 # and the versions available are defined here:
 # https://cloud.google.com/dataflow/docs/reference/flex-templates-base-images
-FROM gcr.io/dataflow-templates-base/java8-template-launcher-base:20210120_RC00
+FROM gcr.io/dataflow-templates-base/java11-template-launcher-base:20210120_RC00
```

Comment:
Might as well move to 20210419_RC00 per the link in comment while we're doing this